src/components/Agreement/static-agreement.js

```diff
@@ -0,0 +1,65 @@
+export default `
+# Aragon Network Agreement
```

Comment:
I pulled this straight from the mock, it's most definitely incorrect or going to change.